apps/updatenotification/src/components/UpdateNotification.vue

```diff
@@ -221,6 +222,7 @@ export default {
 			hideAvailableUpdates: true,
 			openedWhatsNew: false,
 			openedUpdateChannelMenu: false,
+			selectLabel: 'select-input-label-update-settings',
```

Comment:
To not call it a `label`, because it is an `id` of `input`, not of the `label` or a label itself.

```suggestion
			selectId: 'notify-members-settings-select-input',
```

Reply:
But I'd set it directly in the template. It is used only on 2 close lines in the template, not generated, not a part of component data (doesn't define reactive state of the component).